README.md

```diff
@@ -1 +1,16 @@
-# vscode-dvc-pack
\ No newline at end of file
+# DVC Extension Pack
+
+This extension pack packages the most useful extensions for getting started with [DVC](https://dvc.org/).
+
+## Extensions Included
+
+- [DVC for VS Code](https://marketplace.visualstudio.com/items?itemName=iterative.dvc) - Run, compare, visualize, and track machine learning experiments right in VS Code. This extension uses DVC, an open-source data versioning and ML experiment management tool. No additional services or databases are required.
+- [Python](https://marketplace.visualstudio.com/items?itemName=ms-python.python) - Provides rich support for the Python language, including features such as virtual environment support, IntelliSense (Pylance), linting, debugging, code navigation, code formatting, refactoring, variable explorer, test explorer, and more!
+- [YAML](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml) - Provides comprehensive dvc.yaml Language support, via the yaml-language-server.
+- [Markdown Preview Mermaid Support](https://marketplace.visualstudio.com/items?itemName=bierner.markdown-mermaid) - Enables DVC for VS Code to correctly render pipeline DAGs in VS Code's builtin Markdown preview.
```

Comment:
Shouldn't builtin be built-in? 
```suggestion
- [Markdown Preview Mermaid Support](https://marketplace.visualstudio.com/items?itemName=bierner.markdown-mermaid) - Enables DVC for VS Code to correctly render pipeline DAGs in VS Code's built-in Markdown preview.
```

Reply:
Oh, I see that the extension page itself uses *builtin*. Either a typo on their end or I'm wrong 😄